docs/docusaurus/docs/deployment_patterns/how_to_use_gx_with_aws/components/_checkpoint_create_and_run.md

```diff
@@ -1,13 +1,13 @@
 import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
 
-Here we will create and store a <TechnicalTag tag="checkpoint" text="Checkpoint"/> for our Batch, which we can use to validate and run post-validation <TechnicalTag tag="action" text="Actions" />.
+To validate and run post-validation <TechnicalTag tag="action" text="Actions" />, you create and store a <TechnicalTag tag="checkpoint" text="Checkpoint"/> for your Batch.
 
-Checkpoints are a robust resource that can be preconfigured with a Batch Request and Expectation Suite or take them in as parameters at runtime.  They can also execute numerous Actions based on the Validation Results that are returned when the Checkpoint is run.
+Checkpoints can be preconfigured with a Batch Request and Expectation Suite, or they can take them in as parameters at runtime. They can also execute numerous Actions based on the Validation Results that are returned when Checkpoint runs.
 
-This guide will demonstrate using a `SimpleCheckpoint` that takes in a Batch Request and Expectation Suite as parameters for the `context.run_checkpoint(...)` command.
+You'll learn how you can use a `SimpleCheckpoint` that takes in a Batch Request and Expectation Suite as parameters for the `context.run_checkpoint(...)` command.
```

Comment:
Simple Checkpoints are no longer in use -- can we update the examples and surrounding content to use standard Checkpoints?

Reply:
@Rachel-Reverie this issue has been addressed.